lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -2899,6 +2899,12 @@ def visit_FLOAT(self, type_, **kw):
     def visit_DOUBLE_PRECISION(self, type_, **kw):
         return "DOUBLE PRECISION"
 
+    def visit_DOUBLE(self, type_, **kw):
+        return "DOUBLE PRECISION"
+
+    def visit_double(self, type_, **kw):
+        return "DOUBLE PRECISION"
```

Comment:
use the existing visit_DOUBLE_PRECISION method here, return self.visit_DOUBLE_PRECISION

Reply:
Updated